extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -704,14 +853,36 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 				response.databaseMigration.properties.sourceDatabaseName = this._migrationDbs[i];
 				response.databaseMigration.properties.backupConfiguration = requestBody.properties.backupConfiguration!;
 				if (response.status === 201 || response.status === 200) {
+
+					sendSqlMigrationActionEvent(
+						TelemetryViews.MigrationWizardSummaryPage,
+						TelemetryAction.StartMigration,
+						{
+							'hashedServerName': hashString(this._assessmentApiResponse.assessmentResult.name),
+							'hashedDatabaseName': hashString(this._migrationDbs[i]),
+							'migrationMode': requestBody.properties.autoCutoverConfiguration ? 'online' : 'offline',
+							'sessionId': this._sessionId,
+							'migrationStartTime': new Date().toString(),
+							'targetDatabaseName': this._targetDatabaseNames[i],
+							'serverName': this._targetServerInstance.name,
+							'tenantId': this._azureAccount.properties.tenants[0].id,
+							'location': this._targetServerInstance.location,
+							'sqlMigrationService': this._sqlMigrationService.id,
+						},
+						{
+							'irRegistered': this._nodeNames.length > 0 ? 1 : 0
```

Comment:
What does `irRegistered` mean? If this is just a boolean value it should probably go in the properties bucket